packages/nimble-components/src/table-column/text/tests/table-column-text.spec.ts

```diff
@@ -123,21 +123,60 @@ describe('TableColumnText', () => {
     });
 
     const fieldValues = [
-        { dataValue: 'foo', renderedValue: 'foo' },
-        { dataValue: '<button></button>', renderedValue: '<button></button>' },
-        { dataValue: 'null', renderedValue: 'null' },
-        { dataValue: 'undefined', renderedValue: 'undefined' }
+        { value: 'foo' },
+        { value: '<button></button>' },
+        { value: 'null' },
+        { value: 'undefined' },
+        { value: 'null' },
+        { value: '-2147483648/-1' },
+        { value: '田' },
+        { value: 'Ω' },
+        { value: '( ͡° ͜ʖ ͡°)' },
+        { value: '😍' },
+        { value: 'Iñtërnâtiônàlizætiøn☃💩' },
+        { value: '１' }
```

Comment:
lets throw in `'NaN'`, `'-Infinity'`, `'Infinity'`
also curious how `'\x00'` behaves. you can place null characters in the DOM, shouldn't be an issue.